### PR TITLE
Add first version of Orbit Interlock Control Window

### PR DESCRIPTION
**Pull request by anacso17**
Take a look: 

- Main window:
Screenshot from 2022-03-18 16-03-45
Screenshot from 2022-03-18 14-19-21

- BPM interlock enable control window:
Screenshot from 2022-03-18 17-49-17

- BPM interlock threshold setpoint window:
Screenshot from 2022-03-18 18-19-49
Screenshot from 2022-03-18 18-19-43

- Individual BPM interlock detail window
Screenshot from 2022-03-16 11-15-52
Screenshot from 2022-03-18 16-32-18

### CODE REVIEW

**Review by murilobalves — APPROVED** [review]

File: pyqt-apps/siriushla/si_ap_orbintlk/base.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    CONV_NM2M = 1e-9  # [nm] --> [m]
+    CONV_NM2UM = 1e-3  # [nm] --> [um]
+    CONV_UM2NM = 1e+3  # [um] --> [nm]
+
+    BPM_NAMES = BPMSearch.get_names({'sec': 'SI', 'dev': 'BPM'})
+    BPM_NICKNAMES = BPMSearch.get_nicknames(BPM_NAMES)
+    BPM_POS = BPMSearch.get_positions(BPM_NAMES)
+
+    DOWN_2_UP = {
```
Comment (xresende) [inline pyqt-apps/siriushla/si_ap_orbintlk/base.py:18]: DOWN_2_UP, UP_2_DOWN inverted?

**Review by xresende — APPROVED** [review]

File: pyqt-apps/siriushla/si_ap_orbintlk/base.py (hunk 1 above)
Comment (anacso17) [reply pyqt-apps/siriushla/si_ap_orbintlk/base.py]: fixed and migrated to siriuspy

File: pyqt-apps/siriushla/si_ap_orbintlk/custom_widgets.py [hunk 2]
```diff
@@ -362,7 +362,7 @@ def _setupUi(self):
                 text = '\n\nBPM calculation consider the angulation\n' \
                     'estimative:' \
                     '\n\n(pos @downstream - pos @upstream)\n\n' \
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/si_ap_orbintlk/custom_widgets.py:365]: I think it would look better without this space between `pos` and `@`

Comment (anacso17) [reply pyqt-apps/siriushla/si_ap_orbintlk/custom_widgets.py]: done!

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**